src/Simulation/Simulators.Tests/RuntimeMetadataTests.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please add more tests for the Partial, Adjoint+Adjoint, Controlled, and user-defined operations.